### PR TITLE
Added SM-J320F as Galaxy J3 (Duos)

### PR DESCRIPTION
**Pull request by negram**
Also added YaBrowser v17.10

### CODE REVIEW

File: resources/user-agents/browsers/android-webview/android-webview-4-0.json [hunk 1]
```diff
@@ -1107,6 +1107,17 @@
             "Android_4_4"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*)*applewebkit*(*khtml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Chrome*Mobile*Safari*",
```
Comment (jaydiablo) [inline resources/user-agents/browsers/android-webview/android-webview-4-0.json]: You should be able to remove  the "Mobile" keyword here.

File: resources/user-agents/browsers/android-webview/android-webview-4-0.json [hunk 2]
```diff
@@ -1107,6 +1107,17 @@
...
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_0",
```
Comment (jaydiablo) [inline resources/user-agents/browsers/android-webview/android-webview-4-0.json]: And then move this group where the others with the same `match` are, sorted by Android version.

File: resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json [hunk 3]
```diff
@@ -53,6 +53,17 @@
             "Android_6_0"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```
Comment (jaydiablo) [inline resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json]: This should be moved up above the ones that have `Android_7_0` as the highest android version.

File: resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json [hunk 4]
```diff
@@ -150,6 +150,17 @@
             "Android_5_1", "Android_5_0"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```
Comment (jaydiablo) [inline resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json]: Move this one up as well.

File: resources/user-agents/browsers/android-webview/android-webview-4-0.json (hunk 2 above)
Comment (negram) [reply resources/user-agents/browsers/android-webview/android-webview-4-0.json]: Sorry, move to where?

File: resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json (hunk 3 above)
Comment (negram) [reply resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json]: done

File: resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json (hunk 4 above)
Comment (negram) [reply resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json]: and this one is done

File: resources/user-agents/browsers/android-webview/android-webview-4-0.json (hunk 2 above)
Comment (jaydiablo) [reply resources/user-agents/browsers/android-webview/android-webview-4-0.json]: After this group: https://github.com/browscap/browscap/blob/master/resources/user-agents/browsers/android-webview/android-webview-4-0.json#L70

File: resources/user-agents/browsers/chrome/chrome-android-43-on.json [hunk 5]
```diff
@@ -65,6 +65,17 @@
             "Android_5_1"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Chrome/#MAJORVER#.#MINORVER#*Safari/*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```
Comment (mimmi20) [inline resources/user-agents/browsers/chrome/chrome-android-43-on.json]: Please add Android 7.0 here too (in the same same line as Android 7.1).

File: resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json [hunk 6]
```diff
@@ -30,6 +30,17 @@
             "Android_5_1", "Android_5_0"
           ]
         },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```
Comment (mimmi20) [inline resources/user-agents/browsers/samsung/samsung-android-browser-2-0-to-6-0.json]: Please add Android 7.0 here too (in the same same line as Android 7.1).

File: resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json [hunk 7]
```diff
@@ -19,6 +19,17 @@
         "Version": "#MAJORVER#.#MINORVER#"
       },
       "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) SamsungBrowser/#MAJORVER#.#MINORVER#*Chrome*Safari*",
+          "devices": {
+            "SM-J320F*": "Samsung SM-J320F"
+          },
+          "platforms": [
+            "Android_7_1",
```
Comment (mimmi20) [inline resources/user-agents/browsers/samsung/samsung-android-browser-6-2-on.json]: Please add Android 7.0 here too (in the same same line as Android 7.1).

File: resources/user-agents/browsers/android-webview/android-webview-4-0.json (hunk 2 above)
Comment (negram) [reply resources/user-agents/browsers/android-webview/android-webview-4-0.json]: done

**Pull request merged**